src/sbd-inquisitor.c

```diff
@@ -487,13 +512,6 @@ void inquisitor_child(void)
             } else {
                 cl_log(LOG_WARNING, "Ignoring SIG_PCMK_UNHEALTHY from unknown source");
             }
-		} else if (sig == SIG_IO_FAIL) {
-			s = lookup_servant_by_pid(sinfo.si_pid);
-			if (s) {
-				DBGLOG(LOG_INFO, "Servant for %s requests to be disowned",
-						s->devname);
-				cleanup_servant_by_pid(sinfo.si_pid);
-			}
```

Comment:
I think if we're going to remove the handling for SIG_IO_FAIL, we should remove it altogether from the code (sigaddset() and header file)